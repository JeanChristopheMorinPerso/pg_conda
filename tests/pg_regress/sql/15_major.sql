BEGIN;
CREATE EXTENSION pg_conda;

SELECT conda_major('1.2.3');
SELECT conda_major('a');
SELECT conda_major('1a.2.3');
ROLLBACK;
