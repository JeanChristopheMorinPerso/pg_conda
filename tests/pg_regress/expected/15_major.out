BEGIN;
CREATE EXTENSION pg_conda;
SELECT conda_major('1.2.3');
 conda_major 
-------------
           1
(1 row)

SELECT conda_major('a');
 conda_major 
-------------
           0
(1 row)

SELECT conda_major('1a.2.3');
 conda_major 
-------------
            
(1 row)

ROLLBACK;
